.github/workflows/release-backwards-compatiblity.yml

```diff
@@ -0,0 +1,83 @@
+name: Backwards Compatibility Test
```

Comment:
Would it be helpful for the team if we add some explanation at the top of these actions to help this understand more easily 😅?

Reply:
Yup - will add some comments on this one